server/Dockerfile-db

```diff
@@ -1,7 +1,3 @@
 FROM postgres:9.5
 
-ENV POSTGRES_DB polis-dev
-ENV POSTGRES_PASSWORD oiPorg3Nrz0yqDLE
-ENV POSTGRES_USER postgres
-
-ADD ./postgres/db_setup_draft.sql /docker-entrypoint-initdb.d/
+COPY ./postgres/db_setup_draft.sql /docker-entrypoint-initdb.d/
```

Comment:
Just so I know, why COPY over ADD?

Reply:
ADD isn't deprecated, but COPY came later and is considered best practice. The difference is small but Docker recommends using COPY for almost everything.
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy